manageiq-schema.gemspec

```diff
@@ -15,6 +15,7 @@ Gem::Specification.new do |s|
 
   s.files = Dir["{db,lib}/**/*", "LICENSE.txt", "Rakefile", "README.md"]
 
+  s.add_dependency "ancestry"
```

Comment:
@Fryguy what do you think of this dependency in schema?  I believe requiring ancestry in schema migration was a thing that was possibly too much work to try to avoid.

Reply:
ancestry buys us `build_ancestry_from_parent_id`

Reply:
I'm ok with this being in schema...if it changes later, we'll have to deal with it.

otherwise we have to copy the guts of `build_ancestry_from_parent_id`